src/test/java/com/google/javascript/clutz/partial/missing_base.js

```diff
@@ -0,0 +1,33 @@
+goog.module('mising.extend');
+
+//!! Using class syntax requires default externs but the test runs faster
+//!! without them. Just use @constructor instead.
+
+/**
+ * @constructor
+ * @extends {direct.ref.A}
```

Comment:
is the "direct.ref" bit significant?

Reply:
There was a way to just get a handle of `A` instead of the qualified namespace `direct.ref.A`, at least once I had that issue, and eversince got the habit of making all type qualified. Doesn't hurt.